### PR TITLE
fix: txid || txId

### PR DESCRIPTION
**Pull request by limpbrains**
- remove camel case input.txId
- remove input.amount, use input.value

depends on #6145 #6148

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jminurqdsdsxbfl5digttvd4vi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/570)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zq2qqlnw2w4nfh3k6ksdjdff4y?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/576)

**Comment by Overtorment** [comment]
:eyes:

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jekckeh2gyoypdeyndyse37jo4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/852)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pdulmj5sukaazbqtlxjpmieu6e?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/854)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/z6o5llabl5v5m6gy7ez6iwvspe?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/855)

**Comment by limpbrains** [comment]
image

**Comment by limpbrains** [comment]
@Overtorment I'm a bit worried that this patch might break existing wallets.

**Comment by Overtorment** [comment]
some tests already failed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ukz2dcyvu6svglyriufttlsvoa?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/894)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xstgzu47iwq4czxg2lg33loyki?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/895)

File: screen/transactions/details.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ const TransactionsDetails = () => {
         {tx.hash && (
           <>
             <View style={styles.rowHeader}>
-              <BlueText style={[styles.txId, stylesHooks.txId]}>{loc.transactions.txid}</BlueText>
+              <BlueText style={styles.txid}>{loc.transactions.txid}</BlueText>
```
Comment (Overtorment) [inline screen/transactions/details.js:272]: no stone left unturned

**Review by Overtorment — APPROVED** [review]
lgtm

**Comment by Overtorment** [comment]
`assert.ok(hd._utxo[0].amount` is left in integration tests

File: screen/transactions/details.js (hunk 1 above)
Comment (limpbrains) [reply screen/transactions/details.js:272]: 🫡

**Comment by limpbrains** [comment]
@marcosrdz Do i need to patch Swift files to, like Transaction.swift has `self.amount = amount` does it come from JS ?

**Comment by marcosrdz** [comment]
> @marcosrdz Do i need to patch Swift files to, like Transaction.swift has `self.amount = amount` does it come from JS ?

Comes from JS. WatchConnectivity.js

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uikxxsvnixmqv22nrpodr35mbq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/898)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/jeghfq7wrr6dcwjma6odronjla?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/900)

**Comment by limpbrains** [comment]
> > @marcosrdz Do i need to patch Swift files to, like Transaction.swift has `self.amount = amount` does it come from JS ?
> 
> Comes from JS. WatchConnectivity.js

Checked, looks safe

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/66jesz6mmdsmbqxvar47nlfi5e?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/912)

**Pull request merged**